exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/work_in_progress/text_ai/txaie_preprocessing.ipynb

```diff
@@ -0,0 +1,1156 @@
+{
```

Comment:
<p>I would just add this as an extra prerequisite step. Running another notebook with GUI from a link looks quite ugly. I was confused by what I saw when I tried that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>let me check, don't understand yet</p><p> </p>